.github/workflows/slash-command-dispatch.yml

```
@@ -0,0 +1,20 @@
+name: Slash Command Dispatch
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  slashCommandDispatch:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Slash Command Dispatch
+        uses: cloudposse/actions/github/slash-command-dispatch@0.12.0
+        with:
+          token: ${{ secrets.GITHUB_BOT_TOKEN }}
```

```suggestion
          token: ${{ secrets.PUBLIC_REPO_ACCESS_TOKEN }}
```

GitHub does not accept any secrets with `GITHUB_` prefix